.github/workflows/multichain-e2e-template.yml

```diff
@@ -115,11 +115,31 @@ jobs:
         working-directory: ./agoric-sdk/multichain-testing
 
       - name: Run @agoric/multichain-testing E2E Tests
-        run: ${{ inputs.test_command }}
-        working-directory: ./agoric-sdk/multichain-testing
+        id: run-tests
+        # These integration tests can be flaky so retry automatically
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 10
```

Comment:
Looks like 10m might have been a tad agressive.

Reply:
Where do you see that? In [this recent job](https://github.com/Agoric/agoric-sdk/actions/runs/14870200223/job/41756688444?pr=10867) the max e2e-tests job was 9m43s. I'll change it to 15.

Reply:
I remember seeing a test that failed because of timeout.